source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -1,4 +1,4 @@
-title: Read the Local Master Key from a File
+title: Read the Locally-managed Master Key from a File
```

Comment:
nit: Locally-Managed

Reply:
Thanks, I was wondering whether hyphenated words would be uppercase in title case.

Reply:
Thanks, I was wondering whether hyphenated words would be uppercase in title case.